src/ContactModels/include/BipedalLocomotionControllers/ContactModels/ContactModel.h

```diff
@@ -0,0 +1,112 @@
+/**
+ * @file ContactModel.h
+ * @authors Giulio Romualdi
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_CONTROLLERS_CONTACT_MODELS_CONTACT_MODEL_H
+#define BIPEDAL_LOCOMOTION_CONTROLLERS_CONTACT_MODELS_CONTACT_MODEL_H
+
+#include <any>
+#include <unordered_map>
+#include <string>
+
+#include <iDynTree/Core/MatrixFixSize.h>
+#include <iDynTree/Core/VectorFixSize.h>
+#include <iDynTree/Core/Wrench.h>
+
+namespace BipedalLocomotionControllers
+{
+namespace ContactModels
+{
+/**
+ * ContactModel is a generic implementation of a contact model. It computes the contact wrench
+ * between the robot and the environments
+ */
+class ContactModel
+{
+protected:
+    iDynTree::Wrench m_contactWrench; /**< Contact wrench between the robot and the environment
+                                         expressed in mixed representation */
+
+    /** Autonomous dynamics of the contact model rate of change (i.e. given a non linear system\f$
+     * \dot{x} = f + g u\f$ the autonomous is \a f */
+    iDynTree::Vector6 m_autonomousDynamics;
+
+    /** Autonomous dynamics of the contact model rate of change (i.e. given a non linear system\f$
+     * \dot{x} = f + g u\f$ the autonomous is \a g */
```

Comment:
This comment seems to be a copy paste?

Reply:
Fixed in 089bce74224f3dc736bc984651332dfa1ef37b5b